lib/monit_interface.py

```diff
@@ -127,3 +134,46 @@ def restart(watch):
 
   logging.info("Restarting watch {0}".format(watch))
   return run_with_retry([MONIT, 'restart', '-g', watch])
+
+def create_monit_config_file(watch, start_cmd, stop_cmd, ports, match_cmd,
```

Comment:
Yes, the way the config file created for apps is somehow different from the ones created for cassandra and zookeeper. In Djinn it does it differently for them, so I did the same as that! Thats why I added a new template too!